src/System.IO.FileSystem/tests/Directory/GetFileSystemEntries_str_str.cs

```diff
@@ -652,19 +784,32 @@ public void SearchPatternLongPath()
 
         [Fact]
         [PlatformSpecific(TestPlatforms.Windows)]  // Search pattern with double dots throws ArgumentException
-        public void WindowsSearchPatternWithDoubleDots()
+        [SkipOnTargetFramework(~TargetFrameworkMonikers.NetFramework)]
```

Comment:
If we're skipping on everything but .NET Framework, we probably don't need the PlatformSpecific attribute, too.

Reply:
Thanks